--- xfsprogs/mkfs/xfs_mkfs.c.orig	2024-08-03 12:24:35.560776023 +0000
+++ xfsprogs/mkfs/xfs_mkfs.c	2024-08-03 12:26:34.046966903 +0000
@@ -4134,14 +4134,14 @@
 			.dirftype = true,
 			.finobt = true,
 			.spinodes = true,
-			.rmapbt = true,
+			.rmapbt = false,
 			.reflink = true,
 			.inobtcnt = true,
 			.parent_pointers = false,
 			.nodalign = false,
 			.nortalign = false,
 			.bigtime = true,
-			.nrext64 = true,
+			.nrext64 = false,
 			/*
 			 * When we decide to enable a new feature by default,
 			 * please remember to update the mkfs conf files.
